gladier/client.py

```diff
@@ -630,6 +639,7 @@ def run_flow(self, flow_input=None, use_defaults=True):
                  f'"{flow["action_id"]}"')
         if flow['status'] == 'FAILED':
             raise gladier.exc.ConfigException(f'Flow Failed: {flow["details"]["description"]}')
+        flow['run_url'] = 'https://app.globus.org/flows/%s/runs/%s' % (flow_id,flow["action_id"])
```

Comment:
It's a bad idea to modify the original automate response. If we want `run_url` to be present, we should ask the Automate folks to add it.